src/lib/ngsi/Scope.h

```diff
@@ -31,7 +31,7 @@
 #include "common/Format.h"
 #include "orionTypes/areas.h"
 
-using namespace orion;
+//using namespace orion;
```

Comment:
It seems that having this defined in Scope.h was polutting (due to chained includes, I guess) the usage of namespaces in mongo list subscription at mongoBackend, thus ambiguety existed between orion::Subscription and ngsiv2::Subscription.

Changing this has caused a cascade of little changes in other files, but I think they are an improvements. It seems it is not a good idea to use "use namespace" in a .h file, after this experience...